### PR TITLE
Create the wardrobe page

### PR DESCRIPTION
**Pull request by Thomaltarix**
I created the wardrobe page.
It allows the coach to select one of its clients and go through its wardrobe.
The goal of this page for the coaches is to help their clients with their outfits.

### CODE REVIEW

File: frontend/src/pages/AppWardrobe.vue [hunk 1]
```diff
@@ -1,13 +1,214 @@
...
+import axios from 'axios';
+
+const selectedCustomer = ref();
+const selectedCustomerImage = ref("");
+const customers = ref([]);
+const customerClothes = ref({
+  hat: [],
+  top: [],
+  bottom: [],
+  shoes: [],
```
Comment (Peralban) [inline frontend/src/pages/AppWardrobe.vue:15]: what is this "," :question:

File: frontend/src/pages/AppWardrobe.vue [hunk 2]
```diff
@@ -1,13 +1,214 @@
...
+      clothe.image = "data:image/png;base64," + clothe.image;
+      if (clothe.type === 'hat/cap') {
+        clothes.hat.push(clothe);
+      } else if (clothe.type === 'top') {
+        clothes.top.push(clothe);
+      } else if (clothe.type === 'bottom') {
+        clothes.bottom.push(clothe);
+      } else if (clothe.type === 'shoes') {
+        clothes.shoes.push(clothe);
+      }
```
Comment (Peralban) [inline frontend/src/pages/AppWardrobe.vue:74]: It can be simplified

File: frontend/src/pages/AppWardrobe.vue (hunk 1 above)
Comment (Thomaltarix) [reply frontend/src/pages/AppWardrobe.vue]: Fixed

File: frontend/src/pages/AppWardrobe.vue (hunk 2 above)
Comment (Thomaltarix) [reply frontend/src/pages/AppWardrobe.vue]: Done

**Review by 6im0n — APPROVED** [review]
good for me !

**Review by Peralban — APPROVED** [review]

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

**Pull request merged**